zproject/backends.py

```diff
@@ -101,6 +101,14 @@ def common_get_active_user(email: str, realm: Realm,
         return None
     return user_profile
 
+def get_number_online_users():
```

Comment:
Let's put this at the bottom of zerver/views/realm.py, and call it `get_user_count_for_realm(request)`

Reply:
Oh, sure, and the new name it's actually better haha.
But one question @rishig : why in the zerver/views/realm.py and not in the zproject/backends.py?

Reply:
That's a complicated question, but basically most of our URLs point to things in the zerver/views/ directory. Some things to look up, that might explain the general framework
* Django views (the django documentation on writing views)
* MVC, or model-view-controller framework

Reply:
Duly noted. I'll do my research here, thanks for the topics :).